chan_dongle.h

```diff
@@ -163,6 +164,9 @@ typedef struct pvt
 	char			cell_id[8];
 	char			sms_scenter[20];
 
+	unsigned int		incoming_sms_index;
+	unsigned int		incoming_sms_inbox[SMS_INDEX_MAX / 32];
```

Comment:
If someone used an SMS_INDEX that is not a multiple of 32, we'd get an out-of-bounds by this. Either document is in the .h file, or do (SMS_INDEX_MAX+31)/32 here (or both).

Reply:
Fixed! Now rounded up.